torch/_torch_docs.py

```diff
@@ -4073,6 +4073,45 @@
 
 """)
 
+add_docstr(torch.randint,
+           r"""
+randint(low=0, high, sizes, out=None, dtype=torch.float32) -> Tensor
+
+Returns a tensor filled with random integers generated uniformly
+between :attr:`low` (inclusive) and :attr:`high` (exclusive).
+
+The shape of the tensor is defined by the variable argument :attr:`sizes`.
+
+Args:
+    low (int, optional): Lowest integer to be drawn from the distribution. Default: 0.
+    high (int): One above the highest integer to be drawn from the distribution.
+    sizes (tuple): a tuple defining the shape of the output tensor.
+    out (Tensor, optional): the output tensor
+    dtype (torch.dpython:type, optional) – the desired type of returned Tensor. Default: torch.float32
```

Comment:
This looks like a typo?

Reply:
Didnt get you? Are you talkin about torch.dpython:type ??

Reply:
yes, is that expected?

Reply:
In ATEN :
`Tensor randint(const Type& dtype, int64_t low, int64_t high, IntList size, Generator* generator)`
Found similar parameter in Reference : [torch.bartlett_window](http://pytorch.org/docs/master/torch.html#torch.bartlett_window) so copied it! 
Not Sure though!

Reply:
the link you posted doesn't render great, so I think it's not right. @SsnL do you know what's the right way of doing it?

Reply:
hmm `bartlett_window` uses `torch.dtype`(https://github.com/pytorch/pytorch/blob/master/torch/functional.py#L281). But that get's incorrectly rendered to this somehow. I'll try to fix it.

Btw, the ` - ` should be a colon.